src/client/repository/RoomsRepository.js

```diff
@@ -29,6 +31,7 @@ RoomsRepository.prototype.attachEvents = function()
     this.client.on('room:close', this.onRoomClose);
     this.client.on('room:players', this.onRoomPlayers);
     this.client.on('room:game', this.onRoomGame);
+    this.client.on('connect:nbPlayers', this.onUpdateNbPlayers);
```

Comment:
Event name are all in lower case so far (I think), better stick to that with `nb_players` (or `total_players` cf the comment above :))